### DISCUSSION TITLE
measuring neutral

### DISCUSSION THREAD

**Discussion by tzapu** [emporia-vue-local/esphome#210] [Q&A] [answered]
i may be wrong, but i thought in a three phase system that has single phase consumers connected between neutral and each phase, you could use a single CT and measure on the nutral wire.

is there any possibility of doing that with this component? i see there is a phase specified for each circuit so i guess not as it is right now, but could it be made to?

example, i ve got an induction stove top connected to two phases and the oven on the third. i don't really care how much they consume individually, as a whole would be enough and would save me two CTs to use for other circuits...

**Comment by clowrey** [comment, marked as answer]
No you cannot tell total power like that with just one CT on neutral on a three phase system. You need 3 CT.

Longer answer = some of the current goes directly from phase to phase. Only the imbalanced current goes through neutral.

**Reply by tzapu** [reply]
so even if all the consumers are single phase consumers, connected between neutral and one of the phases, if i put a CT on it, it will only measure the imbalance between loads, ie for loads of 2kW on phase A and 1kW on phase two, a CT on neutral will only show 1Kw load?

**Reply by tzapu** [reply]
i now understand better, thank you for taking the time to reply
